task/buildpacks/0.2/buildpacks.yaml

```diff
@@ -107,8 +107,8 @@ spec:
         - name: $(params.PLATFORM_DIR)
           mountPath: /platform
       securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
+        runAsUser: $(params.USER_ID)
+        runAsGroup: $(params.GROUP_ID)
```

Comment:
AFAIK this doesn't works as currently Tekton doesn't supports integer type values and only supports `string` and `array` type values whereas `runAsUser` and `runAsGroup` requires integer values.

/cc @vdemeester @sbwsg 
Please correct me if I am wrong :sweat_smile:

Reply:
Yeah… this is something we need to fix at some point

Reply:
Hi @vdemeester ,

Do you mean fix to support integer type or fix like what I did in this PR?

Reply:
I don't know very well how kubernetes behaves here in regard with this task, but what about just setting up : 

`.spec.securityContext.runAsNonRoot`

to `True` 

we would not need those runAsUser and Group anymore, right ?

Reply:
Yes, I think I see the problem:
`cannot unmarshal string into Go struct field SecurityContext.spec.steps.securityContext.runAsGroup of type int64`

I cannot modify it like this way. :(.

I think it cannot be fixed until Tekton fix the limitation.

Sorry for that, I will close this PR in my side.

Reply:
@zhangtbj I think you could just quote it with "" but i like to know if setting runAsNonRoot may be a better solution, unless you really want a fixed uuid

Reply:
Yes, I quote them like:
```
      securityContext:
        runAsUser: "$(params.USER_ID)"
        runAsGroup: "$(params.GROUP_ID)"
```
The same result. Yes, I think we like to fix it but `runAsNonRoot` may be not enough, it should use a dedicated user or group like 1000 in this step.